src/main.ts

```diff
@@ -67,6 +67,28 @@ function setupMenu() {
         { role: 'togglefullscreen' },
       ],
     },
+    {
+      label: '&Settings',
+      submenu: [
+        {
+          label: 'Language',
+          submenu: [
+            {
+              label: 'English',
+              type: 'radio',
+              checked: i18n.language === 'en',
+              click: () => i18n.changeLanguage('en'),
+            },
+            {
+              label: 'Deutsch',
+              type: 'radio',
+              checked: i18n.language === 'de',
+              click: () => i18n.changeLanguage('de'),
+            },
+          ],
+        },
+      ],
+    },
```

Comment:
How about also putting a flag icon somewhere in the UI?

Reply:
Any idea where? If we put it inside the notebook view it looks like the language change is local to that notebook only...

Reply:
Let's revisit later.